solr/solr-ref-guide/modules/deployment-guide/pages/circuit-breakers.adoc

```diff
@@ -72,6 +73,30 @@ To enable and configure the CPU utilization based circuit breaker:
 
 The `threshold` is defined in units of CPU utilization.
 
+== Advanced example
+
+In this example we will prevent update requests above 80% CPU load, and prevent search requests above 95% CPU load.
+This would prevent expensive bulk updates from impacting search. Note also the support for short-form class name.
+
+[source,xml]
+----
+<config>
+  <circuitBreaker class="solr.CPUCircuitBreaker">
+   <double  name="threshold">80</double>
+   <arr name="requestTypes">
+     <str>update</str>
```

Comment:
I would add in the doc that possible values for `requestsTypes `are defined in enum `RequestType`.

Reply:
Done. Since we switched to a more generic Enum class, we needed to validate supported types separately, so I just documented in ref guide the two supported ones.